mx.jruby/suite.py

```diff
@@ -23,14 +23,12 @@
 
     "jruby-truffle" : {
       "subDir" : "",
+      "class" : "MavenProject",
```

Comment:
The `MavenProject` and `MavenBuildTask` approach seems to be working quite well. The next step is to do something with the distribution. @gilles-duboscq, please push me in the right direction.